build/yamls/antrea.yml

```diff
@@ -1367,6 +1367,15 @@ spec:
         app: antrea
         component: antrea-agent
     spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: antrea/instance
+                operator: NotIn
+                values:
+                - simulator
```

Comment:
Is there a way to not affect the main components' affinity?

Reply:
Do you mean not modify agent yaml file?

Reply:
as discusses, I 've moved this section into yaml patch